src/app/shared/components/layout/layout.component.html

```diff
@@ -0,0 +1,26 @@
+<nz-layout class="app-layout">
+  <nz-sider class="menu-sidebar" nzCollapsible nzWidth="256px" nzBreakpoint="md" [(nzCollapsed)]="isCollapsed" [nzTrigger]="null"> 
+    <app-sidebar [isCollapsed]="isCollapsed" />
+  </nz-sider>
+  
+  <nz-layout>
+    <nz-header>
+      <div class="app-header">
+        <span class="header-trigger" (click)="isCollapsed = !isCollapsed">
+          <span class="trigger" nz-icon [nzType]="isCollapsed ? 'menu-unfold' : 'menu-fold'"></span>
+        </span>
+        <div class="right-section">
+          <h5 nz-typography>{{authService.user()?.email}}</h5>
```

Comment:
Let's avoid accessing service's variables from template. Please create variable in the component and use it here